base/core/src/main/java/org/openscience/cdk/stereo/AbstractStereo.java

```diff
@@ -112,6 +112,21 @@ public void setConfigOrder(int cfg) {
         value = getConfigClass() | cfg;
     }
 
+    /**
+     * {@inheritDoc}
+     */
+    public int getGroupInfo() {
+        return value & IStereoElement.GRP_MASK;
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    public void setGrpConfig(int grp) {
```

Comment:
Is `setGroupConfig()` too long?

Reply:
Yes :) but I renamed the other one as knew you’d prefer it that way just forgot to do the setter. Will add